### DISCUSSION TITLE
Mac OS M1 support

### DISCUSSION THREAD

**Discussion by marqueles** [pauleve/docker-mtgo#159] [macOS]
Hi! 
Thank you for the effort on building this tool.

The launch does not work on Macos M1. I guess this is expected.
You can find the logs from docker attached in case this is not expected or is useful.

Thank you again
[logs macos m1 docker-mtgo.txt](https://github.com/pauleve/docker-mtgo/files/9394952/logs.macos.m1.docker-mtgo.txt)
.

**Comment by pauleve** [comment]
Thanks for your feedback. Indeed, there have been several tries on https://github.com/pauleve/docker-mtgo/issues/116, without success so far AFAIK. 
There may be two different resolutions: either making the Docker image work on Mac M1, or wait for Wine to be fully supported on Mac M1 to execute win32 programs, which should happen in a near future, hopefully.

Anyway, do not hesitate to give a try to the attempts in the referenced issue and give feedback there.
